syft/workers/websocket_server.py

```diff
@@ -119,7 +124,15 @@ def __init__(
 
     def start(self):
         """Start the server"""
-
-        start_server = websockets.serve(self._handler, self.host, self.port, max_size=None)
+        # Secure behavior: adds a secure layer applying cryptography and authentication
+        if not (self.cert is None) and not (self.key is None):
```

Comment:
Overall I think this PR looks ok. This boolean is a bit confusing. since we have `secure` in the top, can we use that down here too and `raise` if the settings aren't right. Reading this I get the sense that a user could inadvertently run the server in an insecure mode without realizing it.

Reply:
The secure flag doesn't exist on the server side, but can be implemented if this makes the idea of ​​a secure server more explicit.

**kwargs at client side**
`
kwargs_websocket = {"host": "localhost",
                    "hook": hook,
                    "verbose": True,
                    "secure": True }
`
**kwargs at server side**
`
kwargs = {
    "id": args.id,
    "host": args.host,
    "port": args.port,
    "hook": hook,
    "verbose": True,
    "cert_path" : "localhost.cert",
    "key_path" : "localhost.key"
}
`


Should I add a secure flag to websocket server too?

Reply:
If the server side has the secure flag can we make it reject all non secure connections?

Reply:
By default, the websocket library already rejects connections that do not implement the pre-established security protocol (if it is established). The security protocol is defined when creating the ssl context `ssl_context = ssl.SSLContext(ssl.PROTOCOL_TLS_SERVER)`